src/integrations/optimizely.ts

```diff
@@ -246,6 +246,11 @@ export const isUserInRolloutAudience = (toggleId: ToggleIdType) => {
 
   for (index = 0; index < endIndex; index++) {
     const rolloutRule = config.experimentKeyMap[rollout.experiments[index].key]
+
+    // The method `__checkIfUserIsInAudience()` will return `{result, reasons}` for versions > 4.5.0
```

Comment:
It looks like 4.5.0 does introduce a public API for our hack below, you mentioned you ran into some issues with TypeScript incompatibility, can you elaborate what broke? 

I am ok with a two pronged approach to support your current needs and continue upgrading later, but looks like a move to the latest would be useful at some point.

Reply:
> you mentioned you ran into some issues with TypeScript incompatibility, can you elaborate what broke?

I had several prompts (red lines) on VS Code when I upgraded to the latest version. I haven't checked the issues in depth, so I can't elaborate that much.

I am in favor of moving to the latest version, but I had to skip that during my task to avoid breaking my pre-determined timebox to work on it. Seems like a good option for next ScID.